src/App.jsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
```

Comment:
- [ ] **You Should Never Disable EsLint**
Please enable it back and try to resolve the error.

Reply:
@shuja-shah Please review the topics of using linters, we can disable linters in some situations.